actions/image_partition_action.go

```diff
@@ -80,6 +75,11 @@ GPT sets the partition type to Linux Swap.
 For msdos partition types hex codes see: https://en.wikipedia.org/wiki/Partition_type
 For gpt partition type GUIDs see: https://systemd.io/DISCOVERABLE_PARTITIONS/
 
+- start -- offset from beginning of the disk there the partition starts.
+
+For 'start' and 'end' properties offset can be written in human readable
```

Comment:
It'd be good to duplicate this inline so that both `start` and `end` properties have this.